timur/lib/client/jsx/components/plotter/plotter.jsx

```diff
@@ -85,6 +85,10 @@ const Chart = connect(
   </div>
 );
 
+const accessFilter = (access, documents)=>{
+  return documents.filter(m => m.access == access).sort((a,b) => a > b);
+};
+
```

Comment:
I think I left this in here by accident, I don't think it is used here.

Reply:
Yeh, I forgot to remove it here